app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/Windows.java

```diff
@@ -499,6 +499,8 @@ public void onResume() {
 
         TelemetryWrapper.resetOpenedWindowsCount(mRegularWindows.size(), false);
         TelemetryWrapper.resetOpenedWindowsCount(mPrivateWindows.size(), true);
```

Comment:
@keianhzo Can you help confirm when we will call `onResume()`? When a session restarts?

Reply:
I confirmed it in my local. It will only be called when returning to a new session.

Reply:
We should call `Windows.onResume `:
- When the app is launched for the first time
- When the app is resumed after being paused (home button and resume)
- After a permission prompt is displayed